bin/truffleruby.sh

```diff
@@ -138,13 +138,16 @@ do
     -J:)
         val=-${val:3}
         ;;
-    -J-cp|-J-classpath)
+    -J-cp|-J-classpath|--jvm.cp|--jvm.classpath)
```

Comment:
Is `--jvm.classpath` given as `--jvm.classpath MYJAR` or `--jvm.classpath=MYJAR` ?
If the second one the handling needs to be different (`java` just overrides the previous `-cp` option)

Reply:
yeah good catch, it's with `=`